.github/workflows/build.yml

```
@@ -117,3 +118,24 @@ jobs:
           tags: |
             docker.io/hashicorp/${{env.repo}}:${{env.version}}
             public.ecr.aws/hashicorp/${{env.repo}}:${{env.version}}
+
+  build-docker-ubi-redhat:
+    name: Docker Build UBI Image for RedHat Registry
+    needs:
+      - get-product-version
+      - build
+    runs-on: ubuntu-latest
+    env:
+      repo: ${{github.event.repository.name}}
+      version: ${{needs.get-product-version.outputs.product-version}}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hashicorp/actions-docker-build@v1
+        with:
+          version: ${{env.version}}
+          target: ubi
+          arch: amd64
+          redhat_tag: scan.connect.redhat.com/ospid-60f9fdbec3a80eac643abedf/${{env.repo}}:${{env.version}}-ubi
```

As commented, this tag will need to change to `quay.io/redhat-isv-containers/5f89bbc30b94cf64cfeb500b/${{env.repo}}:${{env.version}}-ubi` on October 20th

---

Red Hat have made their changes live, the tag format will now be as follows (they've dropped the product name from the tags entirely, and use the project ID not the old `ospid-...`):

```suggestion
          redhat_tag: quay.io/redhat-isv-containers/5f89bbc30b94cf64cfeb500b:${{env.version}}-ubi
```

cc @mdeggies

---

Thanks, Sam! I committed your suggestion.